### PR TITLE
JSONDecodeError appears

### PR DESCRIPTION
**Pull request by martinst06**
JSONDecodeError on pico.py

### CODE REVIEW

File: pico.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            elif sensor_type == 'valve':
+                value = "0"
+            else:
+                value = None
+            
+            # item["value"] = value
+            print(value)
+            # values_list.append(item)
+            # send_back(item)
+
```
Comment (cleliameneghin) [inline pico.py:35]: You have many items commented in this PR, do you truly need them for further development?
if no, please remove them.

File: microcontroller.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+def receive_data_and_write_to_tsdb():
+    received_data = SER.read_all().decode("utf-8").strip()
+
+    received_value = json.loads(received_data)
+
+    if received_value == None:
+        print("No value received from MC, somethings wrong")
+    else:
+        print(received_data)
+        # for sensor in received_value:
```
Comment (cleliameneghin) [inline microcontroller.py]: I assume this functionality is still in progress and that is why it is commented out?

File: list.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#     return jsond
+
+
+
+# def main():
+#     read_sensor_list()
+
+# if __name__ == "__main__":
+#     main()
+
```
Comment (cleliameneghin) [inline list.py]: please remove unused methods & add a comment on what functionaliy this file provides

File: microcontroller.py (hunk 2 above)
Comment (martinst06) [reply microcontroller.py]: yes correct. I want to make sure the part before it gets to here works properly

**Pull request closed without merging**